src/pages/integrations/shapeshift/shapeshift-shift/shapeshift-shift.ts

```diff
@@ -42,35 +46,28 @@ export class ShapeshiftShiftPage {
     private shapeshiftProvider: ShapeshiftProvider,
     private translate: TranslateService
   ) {
-    this.walletsBtc = [];
-    this.walletsBch = [];
+    for (const coin of this.currencyProvider.getAvailableCoins()) {
```

Comment:
We should filter ETH for now (until we officially support it on ShapeShift).

Reply:
reverted to master until we support ETH on shapeshift